drivers/sensor/bmi160/bmi160.c

```diff
@@ -251,7 +251,7 @@ struct {
 			      * SENSOR_ATTR_SAMPLING_FREQUENCY attribute.
 			      */
 } bmi160_odr_map[] = {
-	{0,    0  }, {0,     780}, {1,     562}, {3,    120}, {6,   250},
+	{0,    0  }, {0,     781}, {1,     562}, {3,    125}, {6,   250},
```

Comment:
The commit message drops off mid-sentence:
```
There was also some extra conversions from SI->register->SI when
setting the range, this was causing the
```

Reply:
Done